<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>in</class>
 <widget class="QMainWindow" name="in">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>681</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>您的车型</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLineEdit" name="plate"/>
     </item>
     <item row="1" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>您的车牌号</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>入场时间</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="12" column="2">
      <widget class="QPushButton" name="enterBtn">
       <property name="text">
        <string>入场</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>可选择的车位为:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QPushButton" name="returnBtn">
       <property name="text">
        <string>返回主界面</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QComboBox" name="Carsize">
       <item>
        <property name="text">
         <string>小汽车</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>小卡</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>中卡</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大卡</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>航母</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>车的颜色</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;苹方 特粗&quot;;</string>
       </property>
       <property name="text">
        <string>欢迎来到泊车系统！</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QComboBox" name="position"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
